src/hooks/tradingView/useTradingView.ts

```diff
@@ -158,7 +158,14 @@ export const useTradingView = ({
       tvWidgetRef.current = null;
       setIsChartReady(false);
     };
-  }, [selectedLocale, selectedNetwork, !!marketId, hasPriceScaleInfo, orderbookCandlesToggleOn]);
+  }, [
+    tvWidgetRef,
```

Comment:
The tvWidgetRef.current is set within this useEffect. Also does it matter if tvWidgetRef is added to this dep list since it doesn't trigger a re-render?